### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by bmcd**
- Separates out network initialization (geth init, creating docker-compose files, etc) from script generation (start, stop, attach, etc)
- Organizes script generation logic in the generate/scripts folder, for the most part
- Uses these script objects when referencing commands/filenames in user messages
- Adds windows batch script generation as .cmd files
- Checks for OS + Docker availability, disables bash, docker, k8s options accordingly
- No longer checks for minikube exclusively, so any kube cluster will work. (Docker desktop kubernetes works really well actually)
- Replaces docker ip sed command with javascript to work on windows too
- Some fixes for developing/running the tests on windows like normalizing file paths, but there are so many snapshots based on the platform that we will need to do a lot more refactoring to make all the tests pass when running on windows

### CODE REVIEW

File: src/generators/bashHelper.js [hunk 1]
```diff
@@ -1,36 +1,37 @@
-import {
-  getFullNetworkPath,
-} from './networkCreator'
-import {
-  copyFile,
-  libRootDir,
-  writeFile,
-} from '../utils/fileUtils'
+import { getFullNetworkPath } from './networkCreator'
```
Comment (apratt3377) [inline src/generators/bashHelper.js:1]: do we care about dependency cycles? - saying we have one here and with ./transactionmanager

Comment (bmcd) [reply src/generators/bashHelper.js:1]: Yeah there are a few, but they don't seem to cause any issues and i think we can refactor a bit to fix them. Specifically moving the bash script generation out of the bashHelper file. But that was too much for this PR

File: src/generators/scripts/start.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+EXIT_CODE=$?
+
+if [ $EXIT_CODE -ne 0 ];
+then
+  printf "Error: kubectl not found, please install kubectl before running this script.\n"
+  printf "For more information, see our qubernetes project: https://github.com/jpmorganchase/qubernetes\n"
+  exit $EXIT_CODE
+fi
+
+kubectl cluster-info >nul 2>&1
```
Comment (apratt3377) [inline src/generators/scripts/start.js:41]: when i restart docker and don't have minikube running, this command hangs. 
but then when i start up minikube and then stop it, the command is fine.

Comment (bmcd) [reply src/generators/scripts/start.js]: good catch, that's the windows version of that command. I just pushed a fix

**Pull request merged**